loadtest/store/memstore/store.go

```diff
@@ -1119,3 +1119,18 @@ func (s *MemStore) Thread(threadId string) (*model.ThreadResponse, error) {
 	}
 	return nil, ErrThreadNotFound
 }
+
+// PostsWithAckRequests returns IDs of the posts that asked for acknowledgment.
+func (s *MemStore) PostsWithAckRequests() ([]string, error) {
```

Comment:
Just thinking out loud, not asking for a change: Would it make sense to maintain a different list for these types of posts?

Reply:
So far there isn't any explicit need for it. But in future, if needed it surely can be created.

Reply:
Sounds good, thanks!